app/src/test/java/org/simple/clinic/patient/PatientMocker.kt

```diff
@@ -475,4 +475,12 @@ object PatientMocker {
         businessIds = emptyList()
     )
   }
+
+  fun age(
+      ageValue: Int,
+      updatedAt: Instant = Instant.now(),
+      computedDateOfBirth: LocalDate = LocalDate.now(UTC)
```

Comment:
Interestingly, I looked at the code right now and noticed that this property is no longer being used anywhere. All usages of this property have been replaced with `TimeFunctions.estimateAge` and this is basically a useless property that is just there because it needs a migration to remove.

I will mark this property as deprecated for now and this will get removed when we do https://www.pivotaltracker.com/story/show/161671487.

Reply:
Then can we remove this method altogether and use `estimateCurrentAge` instead?

Reply:
Removed the method in favour of just constructing the `Age` directly.